11ty/.env-example

```diff
@@ -0,0 +1,3 @@
+DRUPAL_AUTH=admin:actual-password
+DRUPAL_URL=https://design-system-cms.kubernetes.lib.umich.edu
+DEBUG=false
```

Comment:
Just noting that `DEBUG=false` will still cause `drupal.js` to spew output; it's sort of piggy backing on the 11ty `DEBUG=Eleventy*` mode, which is a _lot_. Shameless green!